client/Lykke.Service.Assets.Client/ReadModels/AssetPairsReadModel.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Autofac;
+using Lykke.Service.Assets.Client.Models.v3;
+using Microsoft.Extensions.Caching.Memory;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Lykke.Service.Assets.Client.ReadModels
+{
+    class AssetPairsReadModel : IAssetPairsReadModel, IStartable
+    {
+        public const string AllKey = "AllAssetPairs";
+
+        private readonly IAssetsService _assetsService;
+        private readonly IMemoryCache _cache;
+
+        public AssetPairsReadModel(IAssetsService assetsService, IMemoryCache cache)
+        {
+            _assetsService = assetsService;
+            _cache = cache;
+        }
+
+        public AssetPair Get(string id)
+        {
+            try
+            {
+                if (!_cache.TryGetValue(id, out AssetPair value))
+                    return null;
+                return value;
+            }
+            catch (System.InvalidCastException)
```

Comment:
We catch this exception to handle passing of the AllKey as id? If so, probably we need to throw?

Reply:
The case for placing try/catch here was when I tested the API and entered AssetId instead of AssetPairId to the swagger.
Since this data can be used for the public API, I think it's better to ignore such errors.

Reply:
Agree